pkg/systemd/services/service-details.jsx

```diff
@@ -222,7 +222,7 @@ export class ServiceDetails extends React.Component {
             unit_properties: {},
             showDeleteDialog: false,
             unitPaths: [],
-            isPinned: this.props.pinnedUnits.includes(this.props.unit.path),
```

Comment:
I did not touch this page since since a long time, but I recall very well, that we used to always work with object-paths instead of unit.Id so that we include unit aliases in the list. However this is a corner case, when these are also pinned, so I am ok with the change.

Reply:
There is no difference. When I pin "dbus-broker.service" on main, the aliased dbus.service appears as pinned as well:

![image](https://user-images.githubusercontent.com/200109/221784669-2fdd21f1-9085-44d1-83fc-c528bc5c23a6.png)

```
> localStorage.getItem('systemd:pinnedUnits')
'["/org/freedesktop/systemd1/unit/dbus_2dbroker_2eservice"]'
```

they are *really* the same objects, with the same object paths.

Reply:
Oh, wait, I see what you mean, sorry!  You meant the other way around. Right, with this branch only the pinned unit would be shown, not the aliased one. So that is a visible feature change indeed. It feels a bit more natural to me that way, but if you think its not, then I can rework this.

I am trying really hard to get rid of the object paths, as they are incredibly expensive.

Reply:
I don't have strong opinion. I totally agree with this change when thinking of the page load cost benefit as well.